bdk-jvm/lib/build.gradle.kts

```diff
@@ -61,12 +62,14 @@ tasks.withType<Test> {
 dependencies {
     implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.1")
```

Comment:
The coroutines library import is required on the Android side as well (`bdk-android/lib/build.gradle.kts`).

Reply:
Should be added